packages/common/src/scopeSupportFacets/python.ts

```diff
@@ -7,10 +7,16 @@ export const pythonScopeSupport: LanguageScopeSupportFacetMap = {
   "name.foreach": supported,
   "name.resource": supported,
   "name.resource.iteration": supported,
+  "name.argument.actual": supported,
```

Comment:
NB: we could extend this to other languages that support keyword arguments like ruby, but we need to have familiarity with them. A quick check of https://github.com/cursorless-dev/cursorless/blob/b62d901a8be2529b537224268ffdd26d9fdd389d/queries/ruby.scm#L4 shows that it doesn't even support argument yet